examples/server/main.go

```diff
@@ -52,7 +52,11 @@ func initConfig() *vcert.Config {
 	}
 	trustBundleFilePath := os.Getenv("TRUST_BUNDLE_PATH")
 	if trustBundleFilePath != "" {
-		buf, err := ioutil.ReadFile(trustBundleFilePath)
+		file, err := os.Open(trustBundleFilePath)
+		if err != nil {
+			panic(err)
+		}
+		buf, err := io.ReadAll(file)
```

Comment:
I think we also need to call Close in `defer`:
```go
defer func() {
    err = file.Close()
}()
```
On the other places where we call .Open as well.